.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+AllCops:
+  TargetRubyVersion: 2.0.0
+
+Documentation:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'features/*'
+    - 'spec/**/*'
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/LineLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 50
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Style/Attr:
+  Exclude:
+    - 'lib/aws-record/record/attributes.rb'
+
+Style/BlockDelimiters:
+  Exclude:
+    - 'spec/**/*'
+
+Style/SymbolArray:
+  EnforcedStyle: brackets
```

Comment:
Why these styles?  I think as much as possible I'd lean towards rubocop defaults.  Or does this syntax break something else?

Reply:
I also prefer defaults (can't thumbs up from mobile)

Reply:
Updated!